### ISSUE TITLE
No focus ring for custom month navigation

### ISSUE THREAD

**Issue by majapw** [airbnb/react-dates#1304]
Currently if you make use of the `navNext` and `navPrev` props, your custom month navigation buttons will not have a focus ring in any browser (and I do not know of a way to overcome this on the consumer side.

@erin-doyle @backwardok

**Comment by erin-doyle** [comment]
I'm _very_ late responding to this, sorry!  But doing some code spelunking it looks like this has been fixed... sort of, here: https://github.com/airbnb/react-dates/commit/d3e12decd3ad18c702a49f71bd7da763a383bbbe.  

I don't know offhand how this _should_ be fixed but I'm not sure I agree with what has been done to handle it.  It feels wrong to be forcing the consumer to add `tabIndex="0"` to their `<div>`s or whatever they may be providing for the `navNext` and `navPrev` props.  The `tabIndex="0"` really correctly belongs on the `<div role="button">` elements but now requires the consumer to figure out that they need to be responsible for adding the `tabIndex` to a non-interactive element as well as having to disable their `eslint-plugin-jsx-a11y` findings when doing so.

But I don't have any other suggestions at this point as to what to do instead.  So you probably can close this unless you want to keep working on figuring something else out.
